go/spec/spec.go

```diff
@@ -247,6 +250,14 @@ func (sp Spec) createDatabase() datas.Database {
 	switch sp.Protocol {
 	case "http", "https":
 		return datas.NewRemoteDatabase(sp.Href(), sp.Options.Authorization)
+	case "aws":
+		u, _ := url.Parse(sp.Href())
+		parts := strings.SplitN(u.Host, ":", 2) // [table] [, bucket]?
+		sess := session.Must(session.NewSession(aws.NewConfig().WithRegion("us-west-2")))
```

Comment:
Hmmm, I wonder if the region should be part of the spec, too. Once we open source this, people might want to run it in a different region.